api-extractor.json

```diff
@@ -0,0 +1,411 @@
+/**
```

Comment:
## [🕵 FluentUIV0 Open the Visual Regressions report to inspect the affected screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-approval/index.html?organization=uifabric&project=fabricpublic&prId=30365&commitId=6210dddab7f4a717c3e1933692ef367571b26c6d&env=prod&clientType=FLUENTUI)


  <details><summary><b>Card</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Card.Card Example Focusable Teams High Contrast.chromium.png|0| Added|

</details>




<div id="vrtCommentfluent-ui_VRT_Pipeline_v0"/>